src/core/mount-setup.c

```diff
@@ -118,6 +118,8 @@ static const MountPoint mount_table[] = {
         { "efivarfs",    "/sys/firmware/efi/efivars", "efivarfs",   NULL,                      MS_NOSUID|MS_NOEXEC|MS_NODEV,
           is_efi_boot,   MNT_NONE                   },
 #endif
+        { "bpf",         "/sys/fs/bpf",               "bpf",        NULL,                      MS_NOSUID|MS_NOEXEC|MS_NODEV,
+          NULL,          MNT_NONE,                  },
```

Comment:
Hmm, I don't get why we would want to mount that unconditionally. Sounds like a normal .automount+.mount combo that can be enabled/disabled at will would be better. If we don't make use of this ourselves, exposing it sounds premature.

Reply:
.automount wouldn't buy us much. iirc this stuff is not built as kmod, hence there's no benefit with lazy loading anything: setting up the automount *and* the bpffs mount unit is going to be more complex, brittle and time consuming than just mounting bpffs right-away.

I played around with adding a new --cgroup-bpf= switch to nspawn (which would allow users to make nspawn use some foreign defined bpf progs), that's why i made this change, and unlikely the new switch this seemed complete and safe enough to already go upstream, in particular as all kinds of downstream code now contains mount() code for this, and I think we should early on provide this out of the box, so that people won't fight for the right mount options in downstream package to use for it.